spring-cloud-task-docs/src/main/asciidoc/batch.adoc

```diff
@@ -68,7 +68,7 @@ public PartitionHandler partitionHandler(TaskLauncher taskLauncher,
 		MavenResource.parse(String.format("%s:%s:%s",
```

Comment:
I think you should show the passing of `MavenProperties` here, with `remoteRepositories` configured (which would typically point to the repo where the custom batch apps are published). Otherwise, this would only be relying upon the local repo.